src/fides/api/ops/graph/config.py

```diff
@@ -491,12 +491,12 @@ class Config:
         arbitrary_types_allowed = True
 
 
-class Dataset(BaseModel):
+class GraphDataset(BaseModel):
```

Comment:
Renamed to not confuse with Fideslang Dataset, which we're using now in "ops" in favor of "FidesopsDataset"

To run a DSR, a Fideslang Dataset (stored in datasetconfig.dataset) is converted into a GraphDataset. Many GraphDatasets make up a DatasetGraph. The DatasetGraph is what we pass to `run_access_request` or `run_erasure_request`